src/gatsby/gatsby-node/pages/createMdxPages.ts

```diff
@@ -225,13 +225,27 @@ const queries = [
         recursive: true,
       });
 
+      // eslint-disable-next-line max-len
+      const selectExcerpt = (description: string, excerpt: string, limit = 200) => {
+        if (description) {
+          const cleanDescription = description.replace(/\n/g, ' ');
+          if (cleanDescription.length <= limit) {
+            return cleanDescription;
+          }
+          return cleanDescription.slice(0, limit) + '…';
+        }
+        const cleanExcerpt = excerpt.replace(/\n/g,' ');
```

Comment:
You don't need to make this a variable since it isn't being used anywhere other than the `return`